grpc-clients/orchestrator.go

```diff
@@ -31,7 +31,10 @@ func NewOcrhestratorClient(opts config.Orchestrator, devMode bool) (*IPFSOrchest
 			grpc.WithTransportCredentials(creds),
 			grpc.WithPerRPCCredentials(dialer.NewCredentials(opts.Key, !devMode)))
 	} else {
-		dialOpts = append(dialOpts, grpc.WithInsecure())
+		dialOpts = append(
+			dialOpts,
```

Comment:
dialOpts should be on the first line I think, since its not the same argument type as the rest (which are part of the spread argument)

```go
dialOpts = append(dialOpts,
    /* other stuff */,
    /* other stuff */)
```